src/test/java/spoon/test/logging/LogTest.java

```diff
@@ -74,7 +74,7 @@ public void testAllLevelsForLogs(Pair<Level, Integer> levelAndExpectedCount) {
 	private static Stream<Pair<Level, Integer>> getLogLevelsAndExpectedCounts() {
 		return Stream.of(
 				Pair.of(Level.DEBUG, 6),
```

Comment:
```suggestion
				Pair.of(Level.DEBUG, 7),
```

Intuitively speaking, should this not be incremented by 1 as some levels of some log statement is replaced by `DEBUG`? I may be wrong. It would be nice to approve workflow run after [`18dbb3d` (#5312)](https://github.com/INRIA/spoon/pull/5312/commits/18dbb3d1c7898b502bea6f506d13bb609653f9a5).